lib/module.cc

```diff
@@ -5,8 +5,11 @@
 #include "process.h"
 #include "memoryjs.h"
 
-uintptr_t module::findModule(const char* moduleName, pid_t processId, const char** errorMessage) {
-    uintptr_t address = 0;
+//TODO: Might be ll on other systems
+#define KLF "l"
```

Comment:
I think you can use `SCNxPTR` from `<cinttypes>`, which should expand to `lx`/`llx`

Reply:
Good call thanks, I initially copied this from [here](https://gitlab.com/procps-ng/procps/-/blob/master/pmap.c) but it makes more sense to use SCNxPTR for this, it's also more expressive.